command/sub8_launch/scripts/tf_republisher.py

```diff
@@ -13,7 +14,11 @@ def got_range(msg):
     if rospy.Time.now() < rospy.Time(0.5):
         listener.clear()
     t = rospy.Time(0)
-    trans, rot = listener.lookupTransform("/base_link", "/map", t)
+    try:
+        listener.waitForTransform('/base_link', '/map', t, rospy.Duration(1))
+        trans, rot = listener.lookupTransform("/base_link", "/map", t)
+    except (tf.Exception, tf.LookupException, tf.ConnectivityException, tf.ExtrapolationException) as e:
```

Comment:
Oh shit, you're right. I mindlessly copy-and-pasted that whole parenthesis. :disappointed: